src/main/java/tw/waterball/judgegirl/cqi/cyclomatic/CyclomaticComplexityCalculatorImpl.java

```diff
@@ -82,6 +83,29 @@ private String callCPreprocessor(String sourceCode) {
         return sourceCode;
     }
 
+    private String removeStringLiterals(String sourceCode) {
+        String result = new String("");
```

Comment:
Use `StringBuilder` to perform string concatenation (will be faster).

basic usage:
```java
StringBuilder result = new StringBuilder();

result.append("a").append('-').append(10);

System.out.println(result.toString());  //the result will be: a-10
```